mne/io/tests/test_apply_function.py

```diff
@@ -63,3 +63,34 @@ def test_apply_function_verbose():
         assert out is raw
         raw.apply_function(printer, verbose=True)
         assert sio.getvalue().count("\n") == n_chan
+
+
+def test_apply_function_ch_access():
+    """Test apply_function is able to access channel idx."""
+
+    def bad_ch_idx(x, ch_idx):
+        """Pass."""
```

Comment:
(optional) If you prefix these functions-within-a-function with underscores to mark them as private, then our style checker won't insist to have this (vacuous) docstring in them.